components/pedigree/resources/src/main/resources/pedigree/model/export.js

```diff
@@ -347,11 +348,14 @@ define([
                  var cancerData = pedigree.GG.properties[i].cancers[cancer];
                  if (!cancerData.affected) {
                      output += "0\t";
+                     if (yob == "0") {
+                        warnMissingDOBUnaff = true;
+                     }
                  } else {
                      var ageAtDetection = cancerData.hasOwnProperty("numericAgeAtDiagnosis") && (cancerData.numericAgeAtDiagnosis > 0)
                                           ? cancerData.numericAgeAtDiagnosis : "AU";
                      output += ageAtDetection.toString() + "\t";
-                     if (yob == "0") {
+                     if (yob == "0" || ageAtDetection == "AU") {
```

Comment:
I think ageAtDetection == "AU" is fine, and is the way to specify cancer for those whose yob is unknown. Moreover, probably it should be ("yob == "0" && ageAtDetection != "AU"), e.g. I think BOADICEA only needs yob when age is given. But need to just test it with real BOADICEA to know for sure